src/main/java/com/openlattice/data/DataApi.java

```diff
@@ -33,46 +31,32 @@
 import java.util.UUID;
 
 public interface DataApi {
-    // @formatter:off
     String SERVICE               = "/datastore";
     String CONTROLLER            = "/data";
     String BASE                  = SERVICE + CONTROLLER;
-    // @formatter:on
 
-    String ASSOCIATION           = "association";
-
-    int    MAX_BATCH_SIZE        = 10_000;
-    String COUNT                 = "count";
+    String ENTITY_SET_ID         = "setId";
+    String SET_ID_PATH           = "/{" + ENTITY_SET_ID + "}";
```

Comment:
Super duper minor, but to keep consistent with other path variable naming, can this be `ENTITY_SET_ID_PATH`?

Reply:
Sure!